low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -243,7 +221,7 @@ def validate(self):
         pass
 
     def process(self):
-        Cmd._call_script(f"{self._script_dir}/{self.script}", self._args)
+        Utility._call_script(f"{self._script_dir}/{self.script}", self._args)
```

Comment:
If _call_script have been moved to cortx utils, then it should be a public method.

Reply:
Done